SpeechResponder/Personality.cs

```diff
@@ -175,17 +176,19 @@ public Personality Copy(string name, string description)
             string iname = name.ToLowerInvariant();
             string copyPath = Constants.DATA_DIR + @"\personalities\" + iname + ".json";
             // Ensure it doesn't exist
-            if (!File.Exists(copyPath))
+            if (File.Exists(copyPath))
             {
-                ToFile(copyPath);
+                return null; // UI for this is the caller's job
```

Comment:
Would it be more appropriate to throw an exception here then to simply return null (perhaps `InvalidOperationException`)?

Reply:
I'd probably go with `ArgumentException` but it's the payload that really matters, especialy if we go with the validation API, which I am leaning towards.